packages/cli/openapi-parser/src/v3/converters/example/ExampleTypeFactory.ts

```diff
@@ -49,6 +49,9 @@ export class ExampleTypeFactory {
     }
 
     private buildFromPartialObjectExample(partialObjectExample: PartialObjectExample): FullExample | undefined {
+        if (Object.keys(partialObjectExample.includedProperties).length === 0) {
+            return undefined;
+        }
```

Comment:
@dsinghvi Added this in for now so we don't produce examples without any properties. We'll want to remove this when we support generating dummy data for excluded properties so that we can create a `FullExample` from a `PartialObjectExample` with zero included properties.